src/main/java/frc/robot/Constants.java

```diff
@@ -104,6 +106,12 @@
     public static final int CLIMBER_CURRENT_LIMIT = 85;
     public static final int CLIMBER_TRIGGER_THRESHOLD = 100;
     public static final double CLIMBER_TRIGGER_THRESHOLD_TIME = 0.5;
+
+    public static final double CLIMBER_RETRACTED_POSITION = 0.0;
+    public static final double CLIMBER_RETRACTED_POSITION_ERROR = 256.0; // PLACEHOLDER
```

Comment:
idk but I feel like calling them "climber (retracted position/extended position/alignment) error" is kind of misleading? Maybe like "acceptable margin of error" would be clearer? I know that's a long name though so however you think of shortening it

Reply:
Used “Acceptable Error” instead